samples/net/sockets/big_http_download/sample.yaml

```diff
@@ -3,7 +3,7 @@ sample:
   name: big_http_download
 tests:
   test:
-    platform_exclude: esp32
+    filter: TOOLCHAIN_HAS_NEWLIBB
```

Comment:
```suggestion filter: TOOLCHAIN_HAS_NEWLIB
```

Reply:
@nashif : Tried and failed to use a beta feature of Github ...

There's a typo. Should be TOOLCHAIN_HAS_NEWLIB

Reply:
```suggestion
    filter: TOOLCHAIN_HAS_NEWLIB
```